### PR TITLE
Fix different small issues in new color schemes

### PR DESCRIPTION
**Pull request by BingoRUS**

### CODE REVIEW

**Review by pavelgruba — DISMISSED** [review]

File: js/viz/themes.js [hunk 1]
```diff
@@ -140,7 +140,6 @@ function patchTheme(theme) {
     mergeScalar(theme.treeMap.group.border, "color", null, theme.axisColor);
     mergeScalar(theme.treeMap.tile.selectionStyle.border, "color", null, theme.primaryTitleColor);
     mergeScalar(theme.treeMap.group.selectionStyle.border, "color", null, theme.primaryTitleColor);
-    mergeScalar(theme.treeMap.group.label.font, "color", null, theme.secondaryTitleColor);
```
Comment (San4es) [inline js/viz/themes.js:143]: Why do we need to remove this assertion? Can we fix it without removing?

File: testing/tests/DevExpress.viz.core/themes.tests.js [hunk 2]
```diff
@@ -250,7 +250,6 @@ QUnit.test("Patched properties on register theme", function(assert) {
     assert.strictEqual(theme.treeMap.group.border.color, theme.axisColor, "treeMap - group.border.color");
     assert.strictEqual(theme.treeMap.tile.selectionStyle.border.color, theme.primaryTitleColor, "treeMap - tile.selectionStyle.border.color");
     assert.strictEqual(theme.treeMap.group.selectionStyle.border.color, theme.primaryTitleColor, "treeMap - group.selectionStyle.border.color");
-    assert.strictEqual(theme.treeMap.group.label.font.color, theme.secondaryTitleColor, "treeMap - group.label.font.color");
```
Comment (San4es) [inline testing/tests/DevExpress.viz.core/themes.tests.js:253]: Can we fix it without removing?

File: js/viz/themes.js (hunk 1 above)
Comment (San4es) [reply js/viz/themes.js:143]: @pavelgruba
LGTY?

File: testing/tests/DevExpress.viz.core/themes.tests.js (hunk 2 above)
Comment (San4es) [reply testing/tests/DevExpress.viz.core/themes.tests.js:253]: @pavelgruba
LGTY?

File: js/viz/themes.js (hunk 1 above)
Comment (BingoRUS) [reply js/viz/themes.js:143]: We used to get the color to the TreeMap group label from the "secondaryTitleColor" constant. But in some new themes, these colors are different. Now you can manually set this color describing a theme.

File: testing/tests/DevExpress.viz.core/themes.tests.js (hunk 2 above)
Comment (BingoRUS) [reply testing/tests/DevExpress.viz.core/themes.tests.js:253]: It doesn't make sense anymore. In this test, we check the patched properties. The TreeMap group label color isn't a patched property anymore (see my comment above).

Comment (pavelgruba) [reply testing/tests/DevExpress.viz.core/themes.tests.js:253]: @San4es Yes, as I understand now there are different variables which values are used for group label. It used to be one single variable, so we 'patched' it in code.

File: js/viz/themes.js (hunk 1 above)
Comment (San4es) [reply js/viz/themes.js:143]: Ok. Thank you for clarification.

File: testing/tests/DevExpress.viz.core/themes.tests.js (hunk 2 above)
Comment (San4es) [reply testing/tests/DevExpress.viz.core/themes.tests.js:253]: Ok

**Review by pavelgruba — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**